app/js/templates/adminTable.hbs

```diff
@@ -50,18 +50,8 @@
                     </td>
                 </tr>
                 {{#unless testNotTaken}}
-                <tr>
-                    <td dataId="{{id}}" dataName="{{name}}" dataEmail="{{email}}"  dataPercentage="{{percentage}}" dataTime="{{time}}" dataTimeAllowed="{{timeAllowed}}" dataCanRetake="{{canRetake}}" dataTestId="{{this.testId}}" dataCategoryId="{{this.categoryId}}">
-                        <button dataId="{{id}}" class="btn download-user-results-button btn_height">Download results</button>
-                    </td>
-                </tr>
                 {{/unless}}
                 {{#unless testNotTaken}}
-                <tr>
-                    <td>
-                       <button dataId="{{id}}" class="btn view-results-button btn_height">View results</button>
-                    </td>
-                </tr>
                 {{/unless}}
```

Comment:
are these handlebar pieces necessary anymore - now they have no content?

When I delete all the {{}} in this section it doesn't seem to change the functionality at all.